### DISCUSSION TITLE
floating invalid

### DISCUSSION THREAD

**Discussion by TJU-CZT** [Anura3D/Anura3D_OpenSource#78] [Q&A] [answered]
Hello everyone, I chose double-point for mulation to simulate the ccolumn collapse, but there will be an error. I have checked it carefully, but I still can't solve the problem. Hope you all can help me.

**Comment by murphy-john** [comment]
Hello @TJU-CZT,
Please read the [posting guidelines](https://github.com/Anura3D/Anura3D_OpenSource/discussions/29#discussion-4274028). You haven't provided enough information to be able to help you.

**Reply by TJU-CZT** [reply, marked as answer]
Thank you
1.During the discussion, someone encountered the same problem as me, as shown in the figure below, but I did not use his solution to solve my problem.
![image](https://user-images.githubusercontent.com/126539502/224478870-c2acc9ad-1eae-430c-8f20-5d56530c7371.png)
2  I don't know how to use the debug feature in Visual Studio to locate the source of the issue in the code,Are you referring to the code under the installation path?I'm not sure which code to debug,when l have a error "floating invalid".
3.I chose double-point for mulation to simulate the ccolumn collapse.
![image](https://user-images.githubusercontent.com/126539502/224478604-a92feb18-0252-4ff5-96e5-f6ded5eaf938.png)
![image](https://user-images.githubusercontent.com/126539502/224478624-2441ca2a-dd41-4d7a-ac39-d2df5b846af7.png)
![image](https://user-images.githubusercontent.com/126539502/224478631-4d7c84b9-9e1f-47fb-88eb-40914a4255fb.png)
![image](https://user-images.githubusercontent.com/126539502/224478649-ef2fe315-a3c9-49ed-8987-e21a1f3c2018.png)
![image](https://user-images.githubusercontent.com/126539502/224478678-dcc0b3e3-ab86-4add-bb08-a944c6791e52.png)
![image](https://user-images.githubusercontent.com/126539502/224478689-39940f42-bc61-4a53-bce9-a6a961e9e3e1.png)
![image](https://user-images.githubusercontent.com/126539502/224478699-84b7e88c-c61a-4bb2-90d2-a978da51000e.png)
![image](https://user-images.githubusercontent.com/126539502/224478707-7e8f8a08-81f4-4974-ae28-f393b9366f10.png)
![image](https://user-images.githubusercontent.com/126539502/224478718-6efe122c-c1ad-46a9-8ca4-370899fbe152.png)
![image](https://user-images.githubusercontent.com/126539502/224478726-ba2775c5-a967-4725-ad5b-39eb05906bec.png)
when l start Initialisation of stresses,it have a error.

Thank you very much for your reply
